rpc-test-client/test/simple-call.js

```diff
@@ -0,0 +1,36 @@
+
+let plasma = require('../index');
+let _ = require('lodash');
+let BigNumber = require('bignumber.js');
+
+
+it('should load client', (done) => {
```

Comment:
We should wrap this in a `describe` to make it clearer which file it comes from.